File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = 0
    w = []
    h = []

    for i in sizes:
        if i[0] < i[1]:
            w.append(i[1])
            h.append(i[0])
        else:
            w.append(i[0])
            h.append(i[1])

    return max(w) * max(h)

File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.java
class Solution {
    public int solution(int[][] sizes) {
        int max_w = 0;
        int max_h = 0;
        
        for(int i=0; i<sizes.length; i++){
            int w = Math.max(sizes[i][0], sizes[i][1]);
            int h = Math.min(sizes[i][0], sizes[i][1]);
            max_w = Math.max(w, max_w);
            max_h = Math.max(h, max_h);
        }
        
        
        
        return max_w * max_h;
        
    }
}